package.json

```diff
@@ -26,14 +26,14 @@
     "node": ">= 5.4.1"
   },
   "dependencies": {
-    "@angular/animations": "6.0.0-rc.0",
-    "@angular/common": "6.0.0-rc.0",
-    "@angular/compiler": "6.0.0-rc.0",
-    "@angular/core": "6.0.0-rc.0",
-    "@angular/forms": "6.0.0-rc.0",
-    "@angular/platform-browser": "6.0.0-rc.0",
+    "@angular/animations": "6.0.0-rc.1",
+    "@angular/common": "6.0.0-rc.1",
+    "@angular/compiler": "6.0.0-rc.1",
+    "@angular/core": "6.0.0-rc.1",
+    "@angular/forms": "6.0.0-rc.1",
+    "@angular/platform-browser": "6.0.0-rc.1",
     "core-js": "^2.4.1",
-    "rxjs": "6.0.0-beta.1",
+    "rxjs": "6.0.0-rc.0",
```

Comment:
@jelbourn I'm not completely sure why Bazel is complaining about not being able to find rxjs. Do you have any ideas?

Reply:
You need to bump the Bazel rules for nodejs and typescript in WORKSPACE. See [here](https://github.com/angular/flex-layout/blob/4864f2b20324e214038bf7cbc57b976ade04e58d/WORKSPACE) for the correct versions

You may also need to add `@rxjs//operators` to some BUILD files if Bazel still complains

Reply:
One more thing, you also need to add "rxjs" to the list of node_modules in the top-level BUILD file. Then the only errors left should be the operators import.

Reply:
After pulling and running locally, the following are the only BUILD files that need to be modified: cdk/a11y, cdk/overlay, and cdk/tree, and only the test sources need `@rxjs//operators` to build correctly

Reply:
Thanks for looking into it, but I'm not sure that's the case. I tried adding it to all of them and it ended up erroring on the `cdk/portal`.

Reply:
You never added rxjs to the list of node_modules in the top-level BUILD

Reply:
You never added rxjs to the top-level BUILD node_modules

Reply:
See [here](https://github.com/angular/flex-layout/pull/652/files#diff-0ea22445d900db774c8deb00f4a4df55R24) for reference

Reply:
Looks like that did it, although it still feels weird to add `rxjs/operators` when it was complaining about `rxjs/index`.

Reply:
The `rxjs/index` issue was resolved by adding the `rxjs` to top-level BUILD and updating the bazel rules. The operators are to reflect the latest changes in structure for RxJS RC